src/Stratis.Bitcoin/Configuration/Logging/NullLogger.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Configuration.Logging
+{
+    /*
```

Comment:
This code is commented out, should this be removed?

Reply:
Yes